app/assets/stylesheets/components/_share-links.scss

```diff
@@ -1,20 +1,21 @@
-$share-button-width: 25px;
-$share-button-height: 25px;
+$share-button-width: 32px;
+$share-button-height: 32px;
 
 .app-c-share-links__link {
   margin-right: $gutter;
-  @include core-16;
+  @include bold-16($line-height: $share-button-height, $line-height-640: $share-button-height);
+  text-decoration: none;
 
-  &:visited .app-c-share-links__link__icon {
-    background-color: $link-visited-colour;
+  &:visited .app-c-share-links__link__icon .background {
+    fill: $link-visited-colour;
```

Comment:
I believe you can reduce all this logic down by using the magic `currentColor` keyword which will inherit the color of the anchor it's nested inside.

Reply:
See comment below.